src/app/home/problem-list/problem-list.component.html

```diff
@@ -1 +1,32 @@
-<p>problem-list works!</p>
+<html>
+  <head>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      th,
+      td {
+        padding: 5px;
+        text-align: left;
+      }
+    </style>
+  </head>
+
+  <body>
+    <table>
+      <thead>
+        <th>Title</th>
+        <th>Description</th>
+      </thead>
+      <tbody>
+        <tr *ngFor="let problem of problems">
+          <td>{{ problem.title }}</td>
+          <td>{{ problem.description }}</td>
+        </tr>
+      </tbody>
+    </table>
```

Comment:
This much is sufficient for the component HTML. The rest will be taken care by the parent component.

Reply:
Updated
[Commit link](https://github.com/armory-uci/vital/pull/8/commits/066d6038cd7ea68ebd6bcc242bd6fed17dc374fb)